style.scss

```diff
@@ -12,6 +11,12 @@ body {
     font-family: Arial, Helvetica, sans-serif;
 }
 
+a {
+    text-decoration: none;
+    border: none;
```

Comment:
Rule for border is not needed, because it's not set by default. So remove it, please.